media/courgette/style.css.php

```diff
@@ -469,3 +469,59 @@
   padding-top: 10px;
   padding-bottom: 10px;
 }
+
+/* ==========================================================================
+   9 = Mail form
+   ========================================================================== */
+
+#ConteneurFormMail {
+  display:flex;
+  justify-content:center;
+}
+
+#form {
+  position: relative;
+  background: #FFFFFF;
+  max-width: 360px;
+  margin: 0 auto 100px;
+  padding: 45px;
+  text-align: center;
+  box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.2), 0 5px 5px 0 rgba(0, 0, 0, 0.24);
+}
+
+form input {
+  background: #f2f2f2;
+  width: 100%;
+  margin: 0 0 15px;
+  padding: 15px;
+  box-sizing: border-box;
+  font-size: 14px;
+}
+
+.Copy {
+  color:red;
+  background-color:black;
+}
+
+form textarea {
+  background: #f2f2f2;
+  width: 100%;
+  margin: 0 0 15px;
+  padding: 15px;
+  box-sizing: border-box;
+  font-size: 14px;
+}
+
+#button_send_mail {
+  text-transform: uppercase;
+  outline: 2px;
+  background: #663d1c;
+  width: 100%;
+  border: 0;
+  padding: 15px;
+  color: #FFFFFF;
+  font-size: 14px;
+  -webkit-transition: all 0.3 ease;
+  transition: all 0.3 ease;
+  cursor: pointer;
+}
```

Comment:
Using existing classes do not work ?

Reply:
I corrected this in part,  not all styles are totally suitable for the form